scripts/snakemake_parse.py

```diff
@@ -134,6 +135,7 @@ def get_filename_pattern(self, file_type, level, file_suffix=None):
             "prepare"     : "{method}.h5ad",
             "integration" : "{method}.h5ad",
             "metrics"     : "{method}_{o_type}.csv",
+            "embeddings"  : "{method}_{o_type}.csv",
             "cc_variance" : "{method}_{o_type}.csv"
```

Comment:
a little nerve-wrecking to have the same filename pattern 3 times here. I guess they are however stored in different folders, right? Is that also true for the `"cc_variance"` files here, @mumichae? Would be good to know if we could accidentally overwrite stuff otherwise.

Reply:
I agree that this workflow to determine the filenames is pretty clumsy. I'd suggest a complete refactoring of the pipeline based on my better understanding of snakemake now. This would make everything a lot easier.
But for now we should focus on getting things to run first.

Reply:
The files shouldn't be overwritten, as they are in different directories. Otherwise, snakemake will detect a conflict if multiple rules can create the same file. So nothing to worry about here.